src/page-multi-facility/FacilityRow.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components";
+import Colors from "../design-system/Colors";
+import Loading from "../design-system/Loading";
+
+import CurveChartContainer from "../impact-dashboard/CurveChartContainer";
+
+import { Facility } from "./types"
+
+const markColors = {
+  exposed: Colors.green,
+  fatalities: Colors.black,
+  hospitalized: Colors.lightBlue,
+  hospitalBeds: Colors.red,
+  infectious: Colors.red,
+};
+
+interface Props {
+  facility: Facility;
+}
+
+const FacilityRow: React.FC<Props> = ({ facility }) => {
+  const {
+    name,
+    model_inputs
+  } = facility
```

Comment:
A few quick nits in this file: semi-colon here and on line 27, and missing newline at the end of the file.

Reply:
@jessex - Also, even though we propose `model_inputs` to be under_scored instead of camelCased in the data model design document, I'd like to suggest we use camelCase everywhere to more closely match JSON and JS naming conventions

Reply:
I'm a okay with that. I vaguely thought @itsjon or someone else had chosen that naming convention for some annoying reason but it's not coming to me right now. If we can go to camelCase, let's do it.

Reply:
I recommend you install a Prettier extension in your text editor of choice, if there is one, unless you enjoy getting comments such as these 😉

Reply:
I crave attention...